### PR TITLE
[1LP][RFR] SSUI page provider check tests

### PR DESCRIPTION
**Pull request by digitronik**
### PRT Run
{{pytest: cfme/tests/services/test_dialog_element_in_catalog.py::test_provider_field_should_display_in_vm_details_page_in_ssui -v}}

### CODE REVIEW

File: cfme/services/myservice/ssui.py [hunk 1]
```diff
@@ -175,6 +176,11 @@ class MyServiceVMDetailsView(MyServicesView):
     # TODO: This view needs enhancement by FA owner.
     #  TimelinesChart Widget not supporting completely need improvements as per SSUI.
 
+    # I won't found better way than indexing on this Page.
```
Comment (mshriver) [inline cfme/services/myservice/ssui.py]: Too bad, thanks for at least commenting it.

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 2]
```diff
@@ -55,7 +57,7 @@ def service_dialog(appliance, widget_name):
 
 
 @pytest.fixture(scope="function")
-def catalog_item(appliance, service_dialog, catalog):
+def catalog_item_local(appliance, service_dialog, catalog):
```
Comment (mshriver) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: You actually don't need to change the name of the fixture, pytest will resolve the most local one matching the node's scope.

File: widgetastic_manageiq/__init__.py [hunk 3]
```diff
@@ -5946,3 +5946,17 @@ def messages(self):
         except NoSuchElementException:
             pass
         return result
+
+
+class SSUICardPFInfoStatus(Widget):
```
Comment (mshriver) [inline widgetastic_manageiq/__init__.py]: Consider subclassing `widgetastic.widget.GenericLocatorWidget` when setting up a widget like this, as it takes care of mapping ROOT to the passed locator.

File: cfme/tests/services/test_dialog_element_in_catalog.py (hunk 2 above)
Comment (digitronik) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: Actually `pytest` took local `catalog_item` fixture  instate of `catalog_item` global fixture; when we call global `service_vm`.

**Pull request merged**